domain/consensus/processes/blockbuilder/test_block_builder.go

```diff
@@ -199,7 +199,13 @@ func (bb *testBlockBuilder) BuildUTXOInvalidBlock(parentHashes []*externalapi.Do
 
 	// We use genesis transactions so we'll have something to build merkle root and coinbase with
 	genesisTransactions := bb.testConsensus.DAGParams().GenesisBlock.Transactions
-	header, err := bb.buildUTXOInvalidHeader(parentHashes, genesisTransactions)
+
+	bits, err := bb.difficultyManager.RequiredDifficulty(tempBlockHash)
```

Comment:
I think you meant `StageDAADataAndReturnRequiredDifficulty`

Reply:
No, there's no need to know the DAA score of a block in order to build a UTXO invalid block